### PR TITLE
[dashboard#service_nagigations] fix ajax loading

### PR DESCRIPTION
**Pull request by macejmic**
> Replace this with a quick explanation of what's included here. 

The service navigations widget is not being loaded correctly. 

https://issues.redhat.com/browse/THREESCALE-5511

**Verification steps** 

Screenshot 2020-07-08 at 09 00 38

### CODE REVIEW

File: app/views/provider/admin/dashboards/widgets/_service_navigations.html.slim [hunk 1]
```diff
@@ -1,75 +1,76 @@
-nav.DashboardNavigation
-  ul.DashboardNavigation-list
+div id=widget.id
```
Comment (josemigallas) [inline app/views/provider/admin/dashboards/widgets/_service_navigations.html.slim:1]: What's this for?

Comment (macejmic) [reply app/views/provider/admin/dashboards/widgets/_service_navigations.html.slim:1]: otherwise, the widget content is not being replaced.

**Comment by thomasmaas** [comment]
Seems there are still things missing, like applications, plans?

File: app/views/provider/admin/dashboards/widgets/_service_navigations.html.slim [hunk 2]
```diff
@@ -1,75 +1,76 @@
...
+  nav.DashboardNavigation
```
Comment (josemigallas) [inline app/views/provider/admin/dashboards/widgets/_service_navigations.html.slim:2]:
```suggestion
nav.DashboardNavigation id=widget.id
```

**Comment by josemigallas** [comment]
> Seems there are still things missing, like applications, plans?

That will depend on the user permissions. I see everything in my local env, however the last one looks misplaced:
Screen Shot 2020-07-08 at 09 21 09

File: app/views/provider/admin/dashboards/widgets/_service_navigations.html.slim [hunk 3]
```diff
@@ -1,75 +1,76 @@
...
-                                        title: 'The Integration is out of date, promote!',
-                                        icon_append_name: 'exclamation-triangle'
-          - else
-            = dashboard_navigation_link 'Integration',
-                                        path_to_service(service),
-                                        icon_name: 'wrench'
+        // Active docs
+        - if can?(:manage, :plans)
+          li.DashboardNavigation-list-item
+           = dashboard_collection_link 'ActiveDoc',
```
Comment (josemigallas) [inline app/views/provider/admin/dashboards/widgets/_service_navigations.html.slim:49]: I noticed this line has a wrong indentation too and it causes the anchor to be rendered outside the `li`.
```suggestion
            = dashboard_collection_link 'ActiveDoc',
```

**Comment by Martouta** [comment]
Thanks 🙏

**Pull request merged**